equinox/nn/_linear.py

```diff
@@ -33,6 +35,8 @@ def __init__(
         - `out_features`: The output size. The output from the layer will be a vector
             of shape `(out_features,)`.
         - `use_bias`: Whether to add on a bias as well.
+        - `dtype`: The dtype to use. Defaults to either `jax.numpy.float32` or
```

Comment:
This should also tell the user that it applies to all trainable parameters (not just the weights but also the bias) IMO

Reply:
I have updated this